.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "examples/ansible-examples"]
+	path = examples/ansible-examples
+	url = https://github.com/EarthmanT/ansible-examples.git
```

Comment:
@EarthmanT Why do you need to have .gitmodule ?

Reply:
For use case.